markdown/dbaas.md

```diff
@@ -252,25 +246,97 @@ Once we registered kubernetes cluster we can use it’s name to create DB Cluste
 
 ```bash
 curl -X POST "http://localhost/v1/management/DBaaS/PXCCluster/Create" \ 
-     -H "accept: application/json" -u “admin:admin” -H "Content-Type: application/json" \ 
-     -d "
-{ 
-        \"kubernetes_cluster_name\": \"my_cluster\", 
-        \"name\": \"my-cluster-1\", 
-        \"expose\": true, 
-        \"params\": { \"cluster_size\": 3, \"pxc\": { \"compute_resources\": { \"cpu_m\": 1000, \"memory_bytes\": 2000000000 }, \"disk_size\": 25000000000, \"image\": \"percona/percona-xtradb-cluster:8.0.25-15.1\" }, \"haproxy\": { \"compute_resources\": { \"cpu_m\": 1000, \"memory_bytes\": 2000000000 } } }
+     -H "accept: application/json" \
+     -H "authorization: Basic YWRtaW46YWRtaW4=" \
+     -H "Content-Type: application/json" \ 
+     -d "{ \"kubernetes_cluster_name\": \"my_cluster\", \"name\": \"my-cluster-1\", \"expose\": true, \"params\": { \"cluster_size\": 3, \"pxc\": { \"compute_resources\": { \"cpu_m\": 1000, \"memory_bytes\": 2000000000 }, \"disk_size\": 25000000000, \"image\": \"percona/percona-xtradb-cluster:8.0.25-15.1\" }, \"haproxy\": { \"compute_resources\": { \"cpu_m\": 1000, \"memory_bytes\": 2000000000 } } } }"
+```
+
+### Request parameters
+
+```
+{
+  "kubernetes_cluster_name": "string",
+  "name": "string",
+  "params": {
+    "cluster_size": 0,
+    "pxc": {
+      "image": "string",
+      "compute_resources": {
+        "cpu_m": 0,
+        "memory_bytes": "string"
+      },
+      "disk_size": "string"
+    },
+    "proxysql": {
+      "image": "string",
+      "compute_resources": {
+        "cpu_m": 0,
+        "memory_bytes": "string"
+      },
+      "disk_size": "string"
+    },
+    "haproxy": {
+      "image": "string",
+      "compute_resources": {
+        "cpu_m": 0,
+        "memory_bytes": "string"
+      }
+    }
+  },
+  "expose": true
 }
-"
 ```
 
+
+
+|Parameter|Description|Notes|
+|-----|-----|-----|
+|kubernetes_cluster_name|Kubernetes cluster name|Required|
+|name|PXC cluster name to create|name|
+|cluster_size|Cluster size|Default: 3|
+|image|Docker image name|Default is the recommended version from the Percona's version service|
+|compute_resources.cpu_m|CPU resources millis|Default: 1000|
+|mcompute_resources.memory_bytes|Max memory size in bytes|Default: 2 Gb|
```

Comment:
https://www.percona.com/doc/kubernetes-operator-for-pxc/operator.html#pxc-sidecars-resources-requests-memory

2G ?

Reply:
Yes, we are setting the same defaults we have in the GUI.
https://github.com/percona/pmm-managed/pull/1097/files#diff-7a56b03a49f9c4fd843c3edfc194110f786609bba47b3ec694b844ec0f3aff8cR38-R41

Reply:
yeah, those aren't Gbs:
`(Note that 1024 = 1Ki but 1000 = 1k; I didn't choose the capitalization.)`

Reply:
https://en.wikipedia.org/wiki/Gigabyte looks like 2GB is a correct value.

Reply:
What we should do? Leave Gb since it is the accepted nomenclature?